features/polyTalkPreview/src/styles.css

```diff
@@ -112,6 +114,7 @@
 }
 
 .info-stage h3 {
+    font-family: var(--jost-medium);
```

Comment:
This is already being defined on line 120, what we would need here is also `font-weight: 100`

Reply:
Also or instead?

Reply:
Instead, since font-family was already defined, anyway, works perfectly now, thank you!